week2/homework/src/index.js

```diff
@@ -1,3 +1,78 @@
 'use strict';
 
 // TODO: Write the homework code in this file
+let fs = require ('fs');
+console.log(process.argv);
+
+let args = process.argv.slice(2);
+//console.log(args);
+
```

Comment:
Unused code better to be removed.

Reply:
i always forget that, i will keep it in mind.